server/src/main/java/com/thoughtworks/go/server/service/GoDashboardService.java

```diff
@@ -65,7 +65,7 @@ public void visit(PipelineConfigs group) {
     }
 
     public void updateCacheForPipeline(CaseInsensitiveString pipelineName) {
-        if (!Toggles.isToggleOn(Toggles.QUICKER_DASHBOARD_KEY)) {
+        if (featureToggleDisabled()) {
```

Comment:
rename this to`isFeatureToggleDisabled` ?

Reply:
Done. And merged.